product.html

```diff
@@ -0,0 +1,199 @@
+
+    <section class="mx-48 my-10 border-2 shadow space-y-10">
+        <div class="flex justify-between gap-2 p-5">
+            <div>
+                <img src="./img/laptop.webp" alt="">
+                <button class="text-blue-700 w-full text-sm text-center">Open gallery</button>
+            </div class="col-span-1.5">
+            <div class="space-y-4">
+                <p class="font-bold bg-yellow-400 w-40 p-2">Save up to $200.00</p>
+                <h3 class="font-bold text-4xl">Surface Laptop Go</h3>
+                <p class="text-sm">Free upgrade to Windows 11 (available late 2021 into 2022). <a class="text-blue-700">Learn more</a></p>
+                <div><a class="text-blue-700 font-bold mt-4">Wish list</a></div>
+                <p class="text-sm">Make the most of every day with the sleek style, performance, and all-day battery life¹ you need in our lightest Surface Laptop,<br> all at an exceptional value.</p>
+                <div>
+                    <h6 class="font-bold text-sm">Our promise to Surface customers</h6>
+                    <p class="text-sm">Microsoft Store offers extended free returns through January 31, 2022 on Surface products plus free expert help, digital<br> workshops, and remote learning opportunities. Our online associates will provide free personal assistance to help you get the<br> right Surface for your needs.**</p>
+                    <button class="text-blue-700 text-sm">Learn more about promise</button>
+                    <button class="text-blue-700 text-sm ml-5">Book your online appointment</button>
+                </div>
+                <div>
+                    <h6 class="font-bold text-sm">Bundle and save with the Surface Laptop Go Essentials Bundle</h6>
+                    <p class="text-sm">Includes your choice of Surface Laptop Go, Microsoft 365, and Microsoft Complete Protection Plan.</p>
+                    <button class="text-blue-700 text-sm mr-5">Build your bundle</button>
+                </div>
+            </div>
+            <div class="space-y-5">
+                <h3 class="text-3xl">Starting From <b>$549.99</b></h3>
+                <p class="text-sm">Flexible payment options available.</p>
+                <a class="text-blue-700 text-sm">Learn more</a><br>
+                <button class="bg-blue-700 text-white w-full p-2">Configure now</button>
+                <p class="text-sm">Free 2-3 day shipping. Free returns.</p>
+                <p class="text-sm">Buying for your Business? Go to</p>
+                <a class="text-blue-700 text-sm">Surface Laptop Go for Business</a>
+            </div>
+        </div>
+        <div class="bg-gray-200 py-5 px-3">
+            <p class="p-1 text-sm">Special pricing for eligible students, parents, teachers and military. May not be combinable with other offers.</p>
+            <button class="p-1 text-sm text-blue-700">Check your eligibility</button>
+        </div>
+    </section>
+    
+    <section>
+        
+
```

Comment:
can you please try to clean up your codebase by deleting unnecessary empty place